CHANGELOG.md

```diff
@@ -26,10 +27,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   recognized filename patterns, the file patterns `app-*.py`, `app_*.py`, `*-app.py`,
   and `*_app.py` are now considered. However, if the directory contains more than
   one file matching these new patterns, you must provide rsconnect-python with an
-  explicit `--entrypoint` argument.
+  explicit `--entrypoint` argument.\
```

Comment:
nit: typo

Reply:
It took me over a minute to realize what the type is...

```suggestion
  explicit `--entrypoint` argument.
```